routes/fakes.go

```diff
@@ -127,3 +129,10 @@ func fakeOauthConfig() *oauth2.Config {
 		RedirectURL: "https://draupnir.org/redirect",
 	}
 }
+
+func NewFakeLogger() (log.Logger, *bytes.Buffer) {
+	var buffer bytes.Buffer
+	buffer.Grow(1024 * 1024)
```

Comment:
Can reduce it, just figured it might be better to have a larger space than you want, given it doesn't autogrow? Also allocating 1MB of space takes all of 1ms or something, so better safe than sorry?

It's kinda hand wavey but if it's super cheap then I'd rather go bigger than smaller I think.

Reply:
It does autogrow.

Reply:
I was totally incorrect here- didn't realise it autogrew. If that's the case I'd like to just remove the grow.